rudder-core/src/main/scala/com/normation/rudder/datasources/DataSourceRepository.scala

```diff
@@ -133,6 +133,16 @@ class DataSourceRepoImpl(
 
   private[this] var datasources = Map[DataSourceId, DataSourceScheduler]()
 
+  // Initialize data sources scheduler, with all sources present in backend
+  getAll match {
```

Comment:
I think it should not be done at class instanciation (but we have to document that the start is not automatic). Not having it hiden in the class instanciation allows to split:

- error from instanciation (most likelly code problem, and hard to debug: "no such class error" most of the time) from error due to boot (most likelly connection pb)
- let the user choose when to trigger datasource fetch (in a future version of Rudder, we may want to boot completly and as fast as possible before starting datasources schedules)